sdk/playwrighttesting/arm-playwrighttesting/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+    
+## 1.0.0-beta.1 (2023-09-22)
```

Comment:
A minor issue,  i found the first PR was created in 9/25, so where is this date coming from?

Reply:
I think this is manual added by @kazrael2119

Reply:
I generated the code on Friday, but pushed them to github on Monday